src/Shared/FileUtilities.cs

```diff
@@ -59,19 +67,16 @@ static internal partial class FileUtilities
         private static void GetTestExecutionInfo()
         {
             // Get the executable we are running
-            var program = Path.GetFileNameWithoutExtension(Environment.GetCommandLineArgs()[0]);
+            var processNameCommandLine = Path.GetFileNameWithoutExtension(Environment.GetCommandLineArgs()[0]);
+            var processNameCurrentProcess = Process.GetCurrentProcess().ProcessName;
```

Comment:
Ahh, I see. I missed the `if (!s_runningTests)` block below.

---
In reply to: [59752231](https://github.com/Microsoft/msbuild/pull/572#discussion_r59752231) [](ancestors = 59752231)